### DISCUSSION TITLE
Easy Improvements for Profile page

### DISCUSSION THREAD

**Discussion by MoOx** [npm/feedback#487] [Website] [answered]
Hi there! I was looking at my profile on npm (https://www.npmjs.com/~moox) and noticed a few things that feels really weird:

1) "packages" tab show a number of "25" maximum, even if you have plenty of packages (in my case around 180). When you load more by scrolling and clicking, the number changes. Maybe you could directly put the right number?

2) The packages are ordered by published date, but with the oldest first! That make your 25 first "packages" not relevant, especially when published a few years ago.

I hope you can improve this things. Feel free to contact me if you want more feedbacks and/or help to make this better.

Have a nice day!

**Comment by ljharb** [comment]
For 1, see https://github.com/npm/feedback/discussions/349

**Comment by MylesBorins** [comment, marked as answer]
FYI engineering is already working on a fix to show the total number of packages rather than the number that have been paginated.

Regarding the order... I need to think on this for a bit and see if there is an easy fix. Thanks for pointing these out!
